docker/test/run-test

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+set -ex
```

Comment:
```suggestion
set -exu
```

Everywhere you want `-e` you usually want `-u`

Reply:
done